packages/primes/primes.1.0.1/url

```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/KitFreddura/OCaml-Primes/archive/1.3.3.tar.gz" 
```

Comment:
The directory says `1.0.1`, but the filename is `1.3.3.tar.gz`.  Could you please rename the directory to match the release number?